test/parallel/test-console.js

```diff
@@ -146,9 +146,6 @@ assert.ok(/^hasOwnProperty: \d+\.\d{3}ms$/.test(strings.shift().trim()));
 assert.strictEqual('Trace: This is a {"formatted":"trace"} 10 foo',
                    errStrings.shift().split('\n').shift());
 
-assert.strictEqual(strings.length, 0);
```

Comment:
Same which mentioned before.

And `assert.strictEqual(strings.length, 0)` equals above `assert.strictEqual(strings.length, process.stdout.writeTimes);`.